internal/jstest/jstest.go

```diff
@@ -53,27 +53,44 @@ func Run(ctx context.Context, dir string, args []string) (*internal.TestResults,
 		TestResults: make(map[string]internal.TestResult),
 	}
 
-	volume := "tests"
-	for _, testName := range files {
-		output, err := runCommand(dir, "mongo", filepath.Join(volume, testName))
-		if err != nil {
-			if _, ok := err.(*exec.ExitError); !ok {
-				return nil, err
+	type item struct {
+		file string
+		err  error
+		out  []byte
+	}
+
+	var volume = "tests"
+
+	ch := make(chan item, len(files))
```

Comment:
```suggestion
	ch := make(chan *item, len(files))
```

To pass only a pointer, not the whole value